lib/auth/init.go

```diff
@@ -130,6 +130,9 @@ type InitConfig struct {
 	// Databases is a service that manages database resources.
 	Databases services.Databases
 
+	// Status is a service that manages cluster status info.
+	Status services.Status
```

Comment:
As I understand it's called like this because later there will be not just alerts managed by this service?

Reply:
Yeah, I couldn't decide on a good existing service to add this too, but IMO having a new service interface for every little method is overkill, so I picked a name that I thought other similar future APIs could be lumped into.